docs/DEVELOPER_GUIDE.md

```diff
@@ -38,7 +38,7 @@ make codegen                     # Create auto-generated YAML files.
 
 ### Build and Deploy
 ```
-make all                                  # build and test code
```

Comment:
You must not be rebased correctly -- this is already in HEAD

Reply:
It is? 

https://github.com/awslabs/karpenter/blob/main/docs/DEVELOPER_GUIDE.md still seems to say make all?